backend/server.js

```diff
@@ -305,41 +311,30 @@ async function onSendToMessengerButtonClick(sender, userPageId, b64ref) {
       existingData.watchingSections = [];
     }
 
-    const wasWatchingClass = existingData.watchingClasses.includes(userObject.classHash);
-
     const sectionWasentWatchingBefore = [];
-
     for (const section of userObject.sectionHashes) {
       if (!existingData.watchingSections.includes(section)) {
         sectionWasentWatchingBefore.push(section);
       }
     }
 
-    const classCode = `${aClass.subject} ${aClass.classId}`;
     // Check to see how many of these classes they were already signed up for.
-    if (wasWatchingClass && sectionWasentWatchingBefore.length === 0) {
-      notifyer.sendFBNotification(sender, `You are already signed up to get notifications if any of the sections of ${classCode} have seats that open up.`);
-    } else if (wasWatchingClass && sectionWasentWatchingBefore.length > 0) {
-      notifyer.sendFBNotification(sender, `You are already signed up to get notifications if seats open up in some of the sections in ${classCode} and are now signed up for ${sectionWasentWatchingBefore.length} more sections too!`);
-    } else if (sectionWasentWatchingBefore.length === 0) {
-      notifyer.sendFBNotification(sender, `Successfully signed up for notifications if sections are added to ${classCode}!`);
-    } else {
-      notifyer.sendFBNotification(sender, `Successfully signed up for notifications for ${sectionWasentWatchingBefore.length} sections in ${classCode}!`);
-    }
 
-    // ok lets add what classes the user saw in the frontend that have no seats availible and that he wants to sign up for
-    // so pretty much the same as courspro - the class hash and the section hashes - but just for the sections that the user sees that are empty
-    // so if a new section is added then a notification will be send off that it was added but the user will not be signed up for it
+    // only auto enrolls if there's one (or less???) sectionz remaining
```

Comment:
yup - one or zero sections. If a class has no sections and the user clicks sign up, they can be notified if sections are added, or some class details change